crates/bevy_ecs/src/change_detection.rs

```diff
@@ -757,7 +753,7 @@ mod tests {
         }
 
         let mut world = World::new();
-        world.last_change_tick = u32::MAX;
+        world.last_change_tick = Tick::new(u32::MAX);
```

Comment:
I would prefer to see this moved into an associated const on Tick: this is quite leaky.

Reply:
There's not really a use for that constant outside of this one test, though.

Reply:
Hmm, fair. Feel free to resolve.